docs/howto/features/ephemeral.md

```diff
@@ -109,6 +69,77 @@ jupyterhub:
       extraVolumeMounts: []
 ```
 
+## (Optional) Sharing `shared` directories from another hub with an ephemeral hub
+
+In some specific cases, we may need to share a `shared` directory from another hub
+on the same cluster with the ephemeral hub. The 'source' hub whose `shared` directory
+we mount may be used to provide common data files, teaching materials, etc for the
+ephemeral hub's users.
+
+1. Setup the [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)
+   in the ephemeral hub's config to point to the same NFS share that the 'source' hub is
+   pointing to, with the following config:
+
+   ```yaml
+   nfs:
+     enabled: true
+     dirsizeReporter:
+       # We don't need to report directory sizes here, as it's already being reported on by
+       # the 'source' hub
+       enabled: false
+     pv:
+       enabled: true
+       mountOptions: <copied-from-source-hub>
+       serverIP: <copied-from-source-hub>
+       baseShareName: <copied-from-source-hub>
+       shareNameOverride: <name-of-source-hub>
+   ```
+
+   A few options should copied from the config of the 'source' hub, and `shareNameOverride`
+   should be set to whatever is the `name` of the 'source' hub in `cluster.yaml`.
+
+   When deployed, this should set up a new PersistentVolume for the ephemeral hub to use
+   that references the same NFS share of the 'source' hub. You can validate this by
+   comparing them:
+
+   ```bash
+   # Get the source hub's NFS volume
+   kubectl get pv <source-hub-name>-home-nfs -o yaml
+   # Get the ephemeral hub's NFS volume
+   kubectl get pv <ephemeral-hub-name>-home-nfs -o yaml
+   ```
+
+   The section under `spec.nfs` should match for both these `PersistentVolume` options.
+
+   ```{note}
+   If you want to learn more about how this is setup, look into `helm-charts/basehub/templates/nfs.yaml`
+   ```
+
+2. Mount *just* the shared directory appropriately:
+
+   ```yaml
+   jupyterhub:
+    singleuser:
+      storage:
+        # We still don't want to have per-user storage
+        type: none
+        extraVolumes:
+          - name: shared-dir-pvc
+            persistentVolumeClaim:
+              # The name of the PVC setup by nfs.yaml for the ephemeral hub to use
+              claimName: home-nfs
+        extraVolumeMounts:
+          - name: shared-dir-pvc
+            mountPath: /home/jovyan/shared
+            subPath: _shared
+            readOnly: true
+   ```
```

Comment:
Indentation fix, adding a space for all lines after 122.

````suggestion
   ```yaml
   jupyterhub:
     singleuser:
       storage:
         # We still don't want to have per-user storage
         type: none
         extraVolumes:
           - name: shared-dir-pvc
             persistentVolumeClaim:
               # The name of the PVC setup by nfs.yaml for the ephemeral hub to use
               claimName: home-nfs
         extraVolumeMounts:
           - name: shared-dir-pvc
             mountPath: /home/jovyan/shared
             subPath: _shared
             readOnly: true
   ```
````

Reply:
I've no idea how you caught that but *great* catch.